syft/formats/spdx22json/to_format_model.go

```diff
@@ -53,32 +55,15 @@ func toPackages(catalog *pkg.Catalog, relationships []artifact.Relationship) []m
 	for _, p := range catalog.Sorted() {
 		license := spdxhelpers.License(p)
 		packageSpdxID := model.ElementID(p.ID()).String()
-		filesAnalyzed := false
```

Comment:
NOTE: `FilesAnalyzed` is _NOT_ related to package checksums, it's related to [Package Verification Code](https://spdx.github.io/spdx-spec/v2.3/package-information/#79-package-verification-code-field)

Reply:
I think `FilesAnalyzed` is generally applicable.
```
Indicates whether the file content of this package has been available for 
or subjected to analysis when creating the SPDX document.
If false, the package shall not contain any files. 
The metadata for the files analyzed field is shown in Table 20.
```

By generating a `checksum` we are confirming that the file content of this package was available/subjected to analysis.

The Cardinality for `Package Verification Code` is
```
0..1 if FilesAnalyzed ([7.8](https://spdx.github.io/spdx-spec/v2.3/package-information/#7.8)) is true or omitted.
```

So `PVC` can be `0 or 1` if `FilesAnalyzed` is true.

`FileAnalyzed` will be true, IF `PVC` is set, but `FilesAnalyzed` being true, has direct dependency on `PVC` being present